L1Trigger/L1THGCal/plugins/be_algorithms/FullModuleSumAlgo.cc

```diff
@@ -71,8 +71,8 @@ void FullModuleSumAlgo<FECODEC,DATA>::run(const l1t::HGCFETriggerDigiCollection&
         l1t::HGCalCluster cluster( reco::LeafCandidate::LorentzVector(), 
                 moduleSum, 0, 0);
         cluster.setModule(moduleId.wafer());
-        cluster.setLayer(moduleId.layer());
-        cluster.setSubDet(moduleId.subdetId());
+        //cluster.setLayer(moduleId.layer());
```

Comment:
Since there is the now a seedDetId in the cluster class, you could maybe set it here with the detId of the module